pycbc/workflow/minifollowups.py

```diff
@@ -732,6 +743,18 @@ def make_singles_timefreq(workflow, single, bank_file, trig_time, out_dir,
             if trig_time in seg:
                 data_seg = seg
                 break
+            elif trig_time == -1.0:
```

Comment:
I think I'm in favour of the `if trig_time == -1.0: DO_NOT_MAKE_A_PLOT` option ... But will approve this code either way, and it can be changed later.